### PR TITLE
Simplify/remove redundant operations.

### PR DESCRIPTION
**Pull request by arturobernalg**

### CODE REVIEW

File: commons-vfs2/src/main/java/org/apache/commons/vfs2/FileSystemConfigBuilder.java [hunk 1]
```diff
@@ -193,7 +193,7 @@ protected Character getCharacter(final FileSystemOptions fileSystemOptions, fina
             if (StringUtils.isEmpty(str)) {
                 return defaultValue;
             }
-            value = Character.valueOf(str.charAt(0));
+            value = str.charAt(0);
```
Comment (garydgregory) [inline commons-vfs2/src/main/java/org/apache/commons/vfs2/FileSystemConfigBuilder.java:196]: Please don't change boxing and unboxing.

**Review by garydgregory — CHANGES_REQUESTED** [review]
See comment which applies to more than 1 change.

File: commons-vfs2/src/main/java/org/apache/commons/vfs2/provider/http/HttpFileSystemConfigBuilder.java [hunk 2]
```diff
@@ -297,7 +297,7 @@ public void setFollowRedirect(final FileSystemOptions opts, final boolean redire
      * @since 2.0
      */
     public void setMaxConnectionsPerHost(final FileSystemOptions opts, final int maxHostConnections) {
-        setParam(opts, HttpConnectionManagerParams.MAX_HOST_CONNECTIONS, Integer.valueOf(maxHostConnections));
+        setParam(opts, HttpConnectionManagerParams.MAX_HOST_CONNECTIONS, maxHostConnections);
```
Comment (garydgregory) [inline commons-vfs2/src/main/java/org/apache/commons/vfs2/provider/http/HttpFileSystemConfigBuilder.java:300]: Leave boxing alone please.

**Comment by arturobernalg** [comment]
done

**Pull request merged**